### DISCUSSION TITLE
Daten kommen nicht mehr im HomeAssistant an

### DISCUSSION THREAD

**Discussion by floi001** [syssi/esphome-jk-bms#379] [Q&A] [answered]
Habe ein seltsames Problem. Seite heute kommen keine Daten mehr im HA an. Ich sehe im Log, das der ESP32 die Daten liest, aber sie erscheinen nicht mehr im HA. Habe das Modul auch schon neu geflasht. Auch ausgetauscht habe ich es schon. Einen neuen Namen und neu Installiert habe ich den ESP32 jetzt mehrere male.  Über MQTT sendet er die Daten. Aber nicht über die API. 
Idee?

**Comment by syssi** [comment]
Kannst du einen Logauszug zur Verfügung stellen? Ich habe die Hoffnung, dass man daraus etwas erkennen kann.

**Reply by floi001** [reply]
[logs_batterie15kwh_logs.txt](https://github.com/syssi/esphome-jk-bms/files/12827462/logs_batterie15kwh_logs.txt)
Danke!

**Comment by floi001** [comment, marked as answer]
Info: Seit gerstern 22:00 funktioniert es wieder. (nach einem Reboot von HA)
Ich vermute, HA hat zu wenig RAM Speicher. Es gab aber keine Meldungen. 
LG
